src/main/java/BankingSystem.java

```diff
@@ -0,0 +1,209 @@
+                    2. Add income
+                    3. Do transfer
+                    4. Close account
+                    5. Log out
+                    0. Exit""");
+
+            String input = scanner.next();
+            System.out.println();
+
+            switch (input) {
+                case "0" -> {
+                    scanner.close();
+                    dao.close();
+                    System.out.println("Bye!");
+                    System.exit(0);
+                }
+                case "1" -> getBalance();
+                case "2" -> addIncome();
+                case "3" -> doTransfer();
+                case "4" -> closeAccount();
+                case "5" -> logOut();
+                default -> System.out.println("You've entered invalid menu item.\n");
+            }
+        }
+    }
+
+    private void getBalance() {
+        System.out.printf("Balance: %d\n", currentAccount.getBalance());
+    }
+
+    private void addIncome() {
+        System.out.println("Enter income:");
+
+        try {
+            int income = Integer.parseInt(scanner.next());
+
+            if (income < 0 || income == 0) {
+                System.out.println("Money can't be a zero or a negative number!\n");
+                return;
+            }
+
+            updateAccount(currentAccount.getCard().number(), income);
+            System.out.println("Income was added!\n");
+        } catch (NumberFormatException e) {
+            System.out.println("You should enter the amount of money you want to add to your balance!\n");
+            addIncome();
+        }
+    }
+
+    private void doTransfer() {
+        System.out.println("Transfer\n" +
+                           "Enter card number:");
+        String recipientCardNumber = scanner.next();
+
+        if (recipientCardNumber.equals(currentAccount.getCard().number())) {
+            System.out.println("You can't transfer money to the same account!\n");
+            return;
+        }
+
+        if (!cardNumberValidation(recipientCardNumber)) {
+            System.out.println("Probably you made a mistake in the card number. Please try again!\n");
+            return;
+        }
+
+        if (!dao.contains(recipientCardNumber)) {
+            System.out.println("Such a card does not exist.\n");
+            return;
+        }
+
+        System.out.println("Enter how much money you want to transfer:");
+
+        try {
+            int moneyToTransfer = Integer.parseInt(scanner.next());
+
+            if (moneyToTransfer < 0 || moneyToTransfer == 0) {
+                System.out.println("Money can't be a zero or a negative number!\n");
+                return;
+            }
+
+            if (moneyToTransfer > currentAccount.getBalance()) {
+                System.out.println("Not enough money!\n");
+                return;
+            }
+
+            dao.update(recipientCardNumber, moneyToTransfer);
+            updateAccount(currentAccount.getCard().number(), -moneyToTransfer);
```

Comment:
Can you put `-moneyToTransfer` in a new variable and call it with the prefix `negate`?

Reply:
Created a new variable negateRemittance and initialized it with negative moneyToTransfer value